src/Sql/AbstractSql.php

```diff
@@ -396,7 +396,7 @@ protected function resolveColumnValue(
             : $this->processInfo['paramPrefix'] . $namedParameterPrefix;
         $isIdentifier         = false;
         $fromTable            = '';
-        if (is_array($column)) {
+        if (is_array($column) && ! empty($column)) {
```

Comment:
Let's please use `[] === ` instead of `empty()`, which is a problematic construct

Reply:
Will do.